### ISSUE TITLE
Need arm64 support

### ISSUE THREAD

**Issue by simonSlamka** [itchio/butler#224]
Hey,

It would be awesome if butler supported arm64. There should be a build available for those of us who use arm64, which is the future.

Thanks

**Comment by cosmotek** [comment]
If you have Go installed, you should be able to install this binary on whatever system you're using. Just make sure to set your $GOBIN to something in your $PATH, and run `go install <repo_url>`

**Comment by graugraugrau** [comment]
Are there any news on this one?

**Comment by aycyang** [comment]
As mentioned in https://github.com/itchio/itch/issues/2925, making an arm64 build of butler available on broth.itch.ovh would unblock releasing an arm64 build of the itch app.

**Comment by cosmotek** [comment]
To clarify my previous comment mentioning `go install <repo_url>`, I brought this up as an alternative option for those on ARM platforms. Rather than waiting for an ARM binary to be released, you should be able to use `go install` to "magically" compile and install the application in your system. Go will detect your platform and OS, and attempt to compile for that target automatically.

**Comment by aycyang** [comment]
Right, that approach is fine if you're using butler directly, but the itch app insists on downloading butler from itch's download server, even if you already have butler installed. It would be better if the itch app checked the system for an existing installation of butler first, but unfortunately it doesn't work that way right now, as far as I can tell. That could certainly change some day.

**Comment by nightm4re94** [comment]
so I recently wrote some GitHub workflows to publish a project on windows, linux, and mac with a build matrix. I was hoping to be able to use butler to then push the artifacts to itch directly from my pipeline, but unfortunately the GitHub hosted mac runners all use ARM64 architecture. This means that for linux and windows, I'm able to publish my artifacts directly to itch, but not for mac. Unfortunately I am completely unfamiliar with butler's tech stack, but I'd love to see someone picking this up!

**Comment by MrPropre** [comment]
> ARM64 builds for Linux and Mac are now published on https://broth.itch.zone/butler

Amazing! I can't wait for the Windows ARM version to be published as well
